binance/client.py

```diff
@@ -2368,7 +2477,7 @@ def get_current_order_count(self):
             ]
 
         """
-        return self._get("rateLimit/order", True)
+        return self._get('rateLimit/order', True, data=params)
```

Comment:
this endpoint does not accept any parameter, so this is not needed

https://binance-docs.github.io/apidocs/spot/en/#query-unfilled-order-count-user_data

Reply:
Binance Documentation says there is a recvWindow:
Old documentation: https://binance-docs.github.io/apidocs/spot/en/#query-unfilled-order-count-user_data
New documentation: https://developers.binance.com/docs/binance-spot-api-docs/rest-api/public-api-endpoints#query-unfilled-order-count-user_data

Reply:
@m-HD Yes, but that's injected automatically. Nonetheless, I think we can keep params to keep the code consistent so don't worry